### ISSUE TITLE
Create Webhook System

### ISSUE THREAD

**Issue by dylanjkotze** [dylanjkotze/zephyr-project-manager#3]
Create webhook system too allow for integrations with Slack, Discord, and other third party apps

**Comment by maguscorvin** [comment]
Please include Flock and Chanty to the chat app.s

**Comment by dylanjkotze** [comment]
Thanks. I will definitely look into adding these two as well and keep you updated.

**Comment by maguscorvin** [comment]
Yes we - use Flock.  Some clients I know use chanty and others are on Slack and Discord.  We ended up on Flock it had more base features for less $.  We didn't need as many add-ons.
